app/models/blueprint.rb

```diff
@@ -4,6 +4,7 @@ class Blueprint < ApplicationRecord
 
   virtual_has_one :bundle
   virtual_has_one :content, :class_name => "Hash"
+  virtual_column :in_use?, :type => :boolean
```

Comment:
@kbrock 👀 are appreciated here to understand how to handle this type of attribute 😄